Introduction/About/task.md

```diff
@@ -0,0 +1,29 @@
+## About the Course
+
+Welcome to the course in basics of Python programming!
+
+#### Why Learn Python?
+Python is one of the most popular programming languages in the world.
+Used extensively in data science, ML, and AI, Python is the fastest-growing major programming language.
+
+It is a perfect starting point for getting acquainted with programming.
+Python is easier to read, write, and learn than most other programming languages.
+With Python you won’t get stuck on a simple beginner’s task.
+Python’s many forums, Q&A platforms, and supportive open-source community are invaluable resources to help keep you motivated as you learn.
+
+#### Course Aims
+
+After completing this course, you should be able to:
+
+- understand and make use of basic programming concepts, such as variables, data structures, functions and classes;
```

Comment:
understand and make use of basic programming concepts, such as variables, data structures, functions, and classes;